task3/task_3_A/main_3_A.js

```diff
@@ -0,0 +1,78 @@
+const downloads = [
+  {
+    id: 1,
+    title: 'Recipe',
+    status: 'Done',
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    status: 'Pending',
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    status: 'Pending',
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    status: 'Pending',
+  },
+  {
+    id: 5,
+    title: 'Books',
+    status: 'Failed',
+  },
+];
+
+const table = document.querySelector('.table');
+const button = document.querySelector('.button');
+
+function createTable() {
+  for (const el of downloads) {
```

Comment:
_el_ is not a good name for variables. Why el has title, name and s.o. Rename please with something more clear.